packages/react-components/source/react/library/card/Card.md

```diff
@@ -9,8 +9,8 @@ import Heading from '../heading';
 import Text from '../text';
 
 const cardExampleStyle = {
-  width: 150,
-  height: 150,
+  minWidth: 150,
```

Comment:
Not sure that the width needs changing (at least in the case of the value report in PE the text reflowed fine WRT width), it's mainly the height that's the issue as the contents will overflow pretty quickly when only the text is re-sized.